src/System.Windows.Extensions/src/System/Drawing/ImageConverter.cs

```diff
@@ -63,5 +80,57 @@ public override PropertyDescriptorCollection GetProperties(ITypeDescriptorContex
         }
 
         public override bool GetPropertiesSupported(ITypeDescriptorContext context) => true;
+
+        private unsafe Stream GetBitmapStream(ReadOnlySpan<byte> rawData)
+        {
+            try
+            {
+                // The data is in the form of OBJECTHEADER. It's an encoded format that Access uses to push imagesinto the DB.
+                OBJECTHEADER pHeader = MemoryMarshal.Read<OBJECTHEADER>(rawData);
+
+                // pHeader.signature will always be 0x1c15.
+                // "PBrush" should be the 6 chars after position 12 as well.
+                if (pHeader.signature != 0x1c15 ||
+                    rawData.Length <= pHeader.headersize + 18 ||
+                    rawData.Slice(pHeader.headersize).IndexOf(PBrush) != 12)
```

Comment:
I wouldn't IndexOf here, just make sure you have enough bytes and slice the "PBrush" and check equality.